src/Jira/Issues/Walker.php

```diff
@@ -277,6 +281,7 @@ public function rewind()
 	 * @return integer The custom count as an integer.
 	 * @link   http://php.net/manual/en/countable.count.php
 	 */
+    #[\ReturnTypeWillChange]
```

Comment:
This kind of stuff would result in an exception/fatal error on PHP versions prior to PHP 8.0. Definitely remove.

Reply:
It does not, as it is a comment here. See the CI results of those versions.
But it is needed in higher PHP versions
As PHPStan correctly points out.

Reply:
We'll see once the tests are fixed.